R/verb-joins.R

```diff
@@ -68,7 +70,8 @@ NULL
 inner_join.tbl_lazy <- function(x, y, by = NULL, copy = FALSE,
                                 suffix = NULL,
                                 auto_index = FALSE, ...,
-                                sql_on = NULL, na_matches = c("never", "na")) {
+                                sql_on = NULL, na_matches = c("never", "na"),
+                               lhs_as = "LHS", rhs_as = "RHS") {
```

Comment:
Use `x_as` and `y_as`  for consistency with first arguments? Or would it be better to have `as = c("LHS", "RHS")`? Or `as_x` and `as_y`?

Reply:
Having only one argument saves a bit of typing but I prefer having two arguments. To me `x_as` and `y_as` read the most natural.

Reply:
I have mild preference towards making the common part of a name the prefix (since it helps with autocomplete), but that doesn't make much difference here. If you prefer `x_as` and `y_as` then we can stick with that.